### PR TITLE
BDRSPS-545: Survey Occurance Data v1.0

### PR DESCRIPTION
**Pull request by chungvl**

### CODE REVIEW

**Review by joecrowleygaia — CHANGES_REQUESTED** [review]
We don't have versioning for SSD yet. Please move all changes to the `survey_occurrence_data/` directory and resubmit.

File: abis_mapping/templates/survey_occurrence_data/schema.json [hunk 1]
```diff
@@ -319,7 +330,18 @@
       "title": "Other Catalog Numbers",
       "description": "...",
       "example": "BHP2012-7521 | M12378",
-      "type": "list",
+      "type": "string",
```
Comment (serge-gaia) [inline abis_mapping/templates/survey_occurrence_data/schema.json:333]: type string but example shows a list type
also in incidental occurrences (V2) otherCatalogNumbers is a list.

Comment (serge-gaia) [reply abis_mapping/templates/survey_occurrence_data/schema.json:333]: There are some mismatches between the schema of incidental and SSD shared fields.
Still merging that but created a ticket for harmonisation of the two schema

**Review by serge-gaia — APPROVED** [review]

**Comment by serge-gaia** [comment]
You need to approve @joecrowleygaia

**Review by joecrowleygaia — APPROVED** [review]

**Pull request merged**